my-express-app/server.js

```diff
@@ -52,46 +52,87 @@ app.get("/videos", async (req, res) => {
   }
 });
 
-
+const difficultyLevels = ["Beginner", "Intermediate", "Hard"];
```

Comment:
One source of improvement for getting the difficulty level and age group level is to get these values straight from the database. That way, if those values from the database ever change you don't need to also update the values here!